### PR TITLE
Work on the conference part of the website

### LINKED ISSUE #39: Why attending conference and why getting involved
(notes taken at a video chat few days ago)

## Personas
- (a) People who spend most of their time programming (classic RSEs)
    - goal: get recognized
    - interest: best practices
    - interest: future career possibilities
- (b) People who spend a few hours per week programming (academic focus but inefficient now)
    - goal: more efficient research
    - interest: solve immediate problems
    - interest: learn about their domain
- (c) PI/Administrators who can improve their institution's research quality
    - goal: learn how to organize their research staff to do research better
    - interest:

### PR DESCRIPTION
**Pull request by bast**
Work in progress. I will not make it perfect but un-draft once the first iteration is in. We can then improve in follow-up iterations. Often easier to change a wrong page then fill an empty page.

### CODE REVIEW

File: conference/covid.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- 2020-10-01 We publish a second update whether we plan an in-person event
+- 2020-10-15 Organizing committee decides about continue or postpone (early-bird registration closes)
+- 2020-10-31 Final go/no-go (registration closes and last possible decision to postpone the conference without venue costs for us)
+
+
+### What happens if we postpone?
+
+- All registrants will be notified by email
+- We will reimburse registration payments
+- We will postpone the event to 2021 (at this point we are not planning an online substitute)
```
Comment (rkdarst) [inline conference/covid.md:33]:
```suggestion
- We will postpone the event to 2021 (at this point we are not planning an online substitute, but our online community-building, including smaller meetings, will continue regardless)
```

File: conference/index.md [hunk 2]
```diff
@@ -11,6 +11,31 @@ still Stockholm.  We will of course continue monitoring the situation in the
...
+    - Their interests:
+      - Best practices
+      - Future career possibilities
+      - Collaboration opportunities
+- **People with research focus who spend one or two hours per week programming**
+    - Their goals:
+      - More efficient research
+    - Their interests:
+      - Solve immediate problems
+      - Learn about their domain
```
Comment (rkdarst) [inline conference/index.md:31]:
```suggestion
      - Learn about best software practices in their domain
```

File: conference/index.md [hunk 3]
```diff
@@ -11,6 +11,31 @@ still Stockholm.  We will of course continue monitoring the situation in the
...
+- **PIs/administrators who can improve their institution's research quality**
+    - Their goals:
+      - Learn how to organize their research staff to do research better
+    - Their interests:
+      - Learn about lessons learned from other groups and institutions
```
Comment (rkdarst) [inline conference/index.md:36]:
```suggestion
      - Learn about best practices in other groups and institutions
      - Come back prepared to improve their own group/institution
```

File: conference/index.md [hunk 4]
```diff
@@ -11,6 +11,31 @@ still Stockholm.  We will of course continue monitoring the situation in the
...
+      - Collaboration opportunities
+- **People with research focus who spend one or two hours per week programming**
+    - Their goals:
+      - More efficient research
+    - Their interests:
+      - Solve immediate problems
+      - Learn about their domain
+- **PIs/administrators who can improve their institution's research quality**
+    - Their goals:
+      - Learn how to organize their research staff to do research better
```
Comment (rkdarst) [inline conference/index.md:34]:
```suggestion
      - Learn how RSEs can complement their existing staff
      - Learn how to nurture a collaborative, educational computational culture
      -  Improve the quality and efficiency of their research
```

File: conference/covid.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+# COVID-19 decision timeline
+
+We in the [organizing committee](/conference/committees/) are and will be very
+careful about minimizing health risk for all participants and those affected by
+participants who travel to the event.
+
+We will be closely monitoring the COVID-19 situation in Sweden and other
+countries so that we can decide whether this event can take place in person
+December 1-2, 2020.
```
Comment (rkdarst) [inline conference/covid.md:13]:
```suggestion
December 1-2, 2020. We will not just consider if we *are allowed to* host
this conference, but if it is *responsible* to host it.
```
This is my own editorial opinion which I accept if others don't agree with.  To review: does this reflect our opinions?

Comment (bast) [reply conference/covid.md:13]: Thank you. Much better wording and for me the responsibility part is number one.

**Comment by bast** [comment]
Merged - I will send the remaining change as smaller pull request (helps me with work to see that there are fewer issues now).

**Pull request merged**